README.md

```diff
@@ -514,7 +514,7 @@ Thus when you create a TODO, it is almost always your name that is given.
      end
      ```
 
-* <a name="no-default-args"></a>Do not use default arguments. Use an options
+* <a name="no-default-args"></a>Do not use default positional arguments. Use named arguments or an options
```

Comment:
The official name is "keyword arguments," they were introduced with Ruby 2.0 and [performance issues with them were fixed in Ruby 2.2](https://bugs.ruby-lang.org/issues/10440), not sure starting from which version our guide should thus recommend their use.